packages/common/src/schema.ts

```diff
@@ -97,13 +138,33 @@ export class Schema {
         const result: any = [];
         for (let i = 0; i < Document.schemaEntries.length; i += 1) {
           const { name, kind } = Document.schemaEntries[i];
-          let value = 'N/A';
+          let value: TProvableSerializationMap[keyof TProvableSerializationMap];
```

Comment:
I think you need to define an alias for this type.

Reply:
fixed